src/credit-card-expiration-field.tsx

```diff
@@ -112,9 +113,6 @@ const CreditCardExpirationField = ({
     const {setFormError, jumpToNext} = useForm();
 
     const validate = (value: ExpirationDateValue, rawValue: string): string | undefined => {
-        if (!rawValue) {
-            return optional ? '' : texts.formFieldErrorIsMandatory || t(tokens.formFieldErrorIsMandatory);
-        }
```

Comment:
I've find this code repeated in most input fields, I've moved it inside `useFieldProps`